<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PressureImageDisplayClass</class>
 <widget class="QMainWindow" name="PressureImageDisplayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PressureImageDisplay</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="LoadFileButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>620</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Pressure File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>590</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ground Truth</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>590</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulation</string>
    </property>
   </widget>
   <widget class="QLabel" name="FileData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>580</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="SimulationData">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>0</y>
      <width>500</width>
      <height>580</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="FrameSlider">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>620</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="FrameNumDisp">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>620</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame:0</string>
    </property>
   </widget>
   <widget class="QLabel" name="FilePath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadSimuBtton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>620</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Simulation File</string>
    </property>
   </widget>
   <widget class="QLabel" name="SimuPath">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>680</y>
      <width>491</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReloadSimuFile">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>620</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload Simulation Frame</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pressureimagedisplay.qrc"/>
 </resources>
 <connections/>
</ui>
